core/templates/dev/head/pages/admin/roles_tab/roles_tab_directive.html

```diff
@@ -0,0 +1,96 @@
+<style>
+    .form-horizontal .control-label {
+        text-align: left;
+    }
+</style>
+
+<div class="container-fluid">
+  <div class="row">
+    <br>
+    <h3 class="text-center text-danger">NOTE: Please don't modify roles from this page yet.</h3>
```

Comment:
This can be used to view roles after migration. Updation from here won't take effect on config lists.
Should I disable the 'update role button' using some constant in app.js like ALLOW_UPDATE_FROM_ROLES_TAB?
Or should I go for disabling the whole roles tab with some constant like ENABLE_ROLES_TAB?